commerce-payment-service/src/main/java/com/liferay/commerce/payment/internal/engine/CommercePaymentEngineImpl.java

```diff
@@ -28,6 +28,7 @@
 import com.liferay.commerce.payment.result.CommerceSubscriptionStatusResult;
 import com.liferay.commerce.payment.service.CommercePaymentMethodGroupRelLocalService;
 import com.liferay.commerce.payment.util.CommercePaymentUtils;
+import com.liferay.commerce.payment.util.comparator.CommercePaymentMethodPriorityComparator;
```

Comment:
Move This comparator to internal package and avoid version change.

Reply:
@marco-leo CommercePaymentMethodPriorityComparator is a comparator class inside the API module of commerca-payment.
We need it to be exported, we can't put it inside the "internal" package